src/app/databaseService.ts

```diff
@@ -225,10 +221,17 @@ function getEvlFeedDetails(
   return evlFeedQueryResults.map((evlFeedQueryResult: EvlFeedData) => evlFeedQueryResult);
 }
 
-async function getEvlFeedByVrm(
-  databaseService: DatabaseServiceInterface,
-  event: EvlEvent,
-): Promise<EvlFeedData[]> {
+function getTflFeedDetails(queryResult: [RowDataPacket[], FieldPacket[]]): TflFeedData[] {
+  const tflFeedQueryResults = queryResult[0] as TflFeedData[];
+
+  if (tflFeedQueryResults === undefined || tflFeedQueryResults.length === 0) {
```

Comment:
Optional: this check is a copy from EVL and can be refactored 😄

Reply:
I think it might be worth creating a generic to be honest, we're likely going to add more of these non-generic functions and may as well do it now whilst the work isn't part of the sprint goal

Reply:
Agree, will look at creating a generic this morning, thanks for this point

Reply:
no problem, while we're at it this duplicated about 90% of the EVL feed ;P